docs/content.zh/docs/connectors/table/kafka.md

```diff
@@ -499,12 +499,13 @@ If `timestamp` is specified, another config option `scan.startup.timestamp-milli
 If `specific-offsets` is specified, another config option `scan.startup.specific-offsets` is required to specify specific startup offsets for each partition,
 e.g. an option value `partition:0,offset:42;partition:1,offset:300` indicates offset `42` for partition `0` and offset `300` for partition `1`.
 
-### Changelog Source
+### CDC Changelog Source
+
+Flink natively supports Kafka as a CDC changelog source. If messages in Kafka topic is change event captured from other databases using CDC tools, then you can use a CDC format to interpret messages as INSERT/UPDATE/DELETE messages into Flink SQL system.
+
+Flink provides three CDC formats: [debezium-json]({% link dev/table/connectors/formats/debezium.md %}), [canal-json]({% link dev/table/connectors/formats/canal.md %}) and [maxwell-json]({% link dev/table/connectors/formats/maxwell.md %}) to interpret change events captured by [Debezium](https://debezium.io/), [Canal](https://github.com/alibaba/canal/wiki) and [Maxwell](https://maxwells-daemon.io/).
```

Comment:
We also support `debezium-avro-confluent` now. The documentation is also located in https://ci.apache.org/projects/flink/flink-docs-master/zh/docs/connectors/table/formats/debezium/

Reply:
Since this list is getting longer why don't we make it an actual list. 

```
Flink provides several CDC formats: 
* [debezium-json]()
* [canal-json]()
* others
```

That way its easier to expand this as we add more formats in the future.

Reply:
Thanks for the feed-back.
How about I make it a list, and I omit the details of the specific flavors of the format, i.e. we choose not to mention here the distinction between `json` and `avro-confluent` and others? That way the user can find all the specific details in the linked pages, and this part of the documentation has less chances of becoming obsolete over time.

Something like this:
```
Flink provides several CDC formats: 
* [debezium](...)
* [canal](...)
* [maxwell](...)
```

What do you think?

Reply:
Sounds good to me!